### PR TITLE
web-i-fication of existing content being distrubited via PDF

### PR DESCRIPTION
**Pull request by mbegan**
## Description:
- This is net new content (as far as developer.okta.com content goes) but a port of an [existing guide](https://www.okta.com/sites/default/files/Okta_Security-Analytics_Partner-Integration-Guide.pdf) 

### Resolves:
* [OKTA-147181](https://oktainc.atlassian.net/browse/OKTA-147181)

Is not listed on the navigation menus (on purpose for now)  the "index" file is called zindex until we can identify a proper merge or delineation of sso/provisioning apps and other advanced integrations.

### CODE REVIEW

**Comment by mbegan** [comment]
@mystiberry-okta would you mind giving this a once over?  quite a bit of content but nothing is really "new" just being ported from a PDF

**Comment by mystiberry-okta** [comment]
Happy to review (tonight), @mbegan! One meta question, any reason not to put this in https://developer.okta.com/use_cases/integrate_with_okta/ ? Or, should we move some of the content in there to /use_cases/isv ?

**Comment by mbegan** [comment]
I went back and fourth on the final resting place and settled with the where i initially started.

I think there is a larger discussion to be had around distinguishing between sso/provisioning (that is covered in https://developer.okta.com/use_cases/integrate_with_okta) and API based integrations that I cover in user_cases/isv, i have some ideas but need to get more input and the outcome wouldn't change the meat of my commit.  This is main reason i didn't include an index I didn't want it to be discoverable and cause confusion.  I will point a few partners to it directly until the delineation between isv integrations becomes clearer.

**Comment by mystiberry-okta** [comment]
Okay, but it is discoverable from google search.

File: _source/_use_cases/isv/embedded-occ.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: docs_page
+title: Embedded OCC
```
Comment (mystiberry-okta) [inline _source/_use_cases/isv/embedded-occ.md]: Spell out the product/feature name on first occurrence and in title.

File: _source/_use_cases/isv/embedded-occ.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+weight: 2
+excerpt: Leverage the power of Okta and Increase Adoption of your SaaS application with Embedded Okta Cloud Connect (OCC)
```
Comment (mystiberry-okta) [inline _source/_use_cases/isv/embedded-occ.md]: Don't capitalize Increase Adoption

File: _source/_use_cases/isv/includes/analytics-appGroups.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+
+The AppGroups endpoint is useful for gaining a better understanding of the meaning of groups present in Okta. For example, risk scores could be inferred based on access granted through a given group.
```
Comment (mystiberry-okta) [inline _source/_use_cases/isv/includes/analytics-appGroups.md]: include link to api doc?

Comment (mystiberry-okta) [reply _source/_use_cases/isv/includes/analytics-appGroups.md]: If possible, can we clean this up a little? i.e.

You can gain a better understanding of the meaning of groups in Okta by looking at the AppGroups endpoint. For example, you can infer risk scores based on the access granted to a user from a given group.

File: _source/_use_cases/isv/includes/analytics-appGroups.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+#### Best Practices appGroups
```
Comment (mystiberry-okta) [inline _source/_use_cases/isv/includes/analytics-appGroups.md]: Call this AppGroups or appGroups everywhere.

Comment (mystiberry-okta) [inline _source/_use_cases/isv/includes/analytics-appGroups.md]: There's no context for this information, it leaps from 10k to ground level details in a single paragraph.

**Comment by mbegan** [comment]
@mystiberry-okta I've incorporated your feedback and also finished the embedded OCC guide port currently hosted in a pdf https://www.okta.com/sites/default/files/Okta_Embedded_OCC_Implementation_Guide_v1.pdf

**Comment by mbegan** [comment]
sigh... travis won't pass until https://github.com/okta/okta.github.io/pull/1509 gets incorporated into weekly...  i don't want to make that a part of my commit, can you advise ?

**Pull request merged**